### PR TITLE
Add NewIntegrationsHandler

### PR DESCRIPTION
**Pull request by ludeeus**
Adds logic to the bot to enforce our requirement of a single platform when adding new integrations.

### CODE REVIEW

File: services/bots/src/github-webhook/handlers/new_integrations.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { PullRequestLabeledEvent } from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+import { fetchPullRequestFilesFromContext } from '../utils/pull_request';
+import { ParsedPath } from '../utils/parse_path';
+
+const REVIEW_COMMENT = `
+When adding new integrations. Limit included platforms to a single platform, use the **Ready for review** button when you are done, thanks :+1:
```
Comment (frenck) [inline services/bots/src/github-webhook/handlers/new_integrations.ts:10]:
```suggestion
When adding new integrations, limit included platforms to a single platform. Please reduce this PR to a single platform and use the **Ready for review** button when you are done, thanks :+1:
```

File: services/bots/src/github-webhook/handlers/new_integrations.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const pullRequestFiles = await fetchPullRequestFilesFromContext(context);
+    const parsed = pullRequestFiles.map((file) => new ParsedPath(file));
+
+    const integrationPlatforms = parsed.filter((path) => path.type === 'platform');
+
+    if (integrationPlatforms.length > 1) {
+      await context.github.pulls.createReview(
+        context.pullRequest({
+          body: REVIEW_COMMENT,
+          event: 'REQUEST_CHANGES',
```
Comment (frenck) [inline services/bots/src/github-webhook/handlers/new_integrations.ts:37]: Wouldn't this also trigger the review drafter rule? (and thus trigger a duplicate comment)

**Review by frenck — CHANGES_REQUESTED** [review]

File: services/bots/src/github-webhook/handlers/new_integrations.ts (hunk 2 above)
Comment (ludeeus) [reply services/bots/src/github-webhook/handlers/new_integrations.ts:37]: Yeah

Comment (ludeeus) [reply services/bots/src/github-webhook/handlers/new_integrations.ts:31]: Removed the drafting from this PR.

**Review by frenck — APPROVED** [review]

**Pull request merged**